ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -125,6 +125,7 @@ open class PillButtonBar: UIScrollView {
 
     private var stackView: UIStackView = {
         let view = UIStackView()
+        view.distribution = .fillProportionally
```

Comment:
why this change?

Reply:
The buttons using the configuration were now the same width by default. The stack view gets the widest button and apply that same width for all the buttons unless the distribution .fillProportionally is set.

Reply:
i see. then shouldn't it be only set for ios15 and newer?

Reply:
I can make it specific for that, but it seems like it'll be just redundant in the worst case scenario for iOS 14. It didn't cause any regressions for iOS 14.
Let me know your preference. I'm fine with either option.

Reply:
i see as long as it didn't cause regression in iOS 14 and the scrolling calculation

Reply:
Yes. I validated on both iOS 14 and 15 and no regressions were introduced by this extra call.